lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -46,6 +46,7 @@ import { Constants,
 } from "./utils/Constants";
 import { CryptoUtils } from "./utils/CryptoUtils";
 import { TrustedAuthority } from "./authority/TrustedAuthority";
+import { server } from 'sinon';
```

Comment:
I dont think you meant to include this.

Reply:
Yes I just noticed, fixing for AuthCache as well.